doc/source/client.rst

```diff
@@ -10,6 +10,9 @@ The client is NOT thread safe, meaning the application must ensure that calls ar
 This is only a problem for synchronous applications that use multiple threads or for asynchronous applications
 that use multiple :mod:`asyncio.create_task`.
 
+The client no longer supports task cancellation, please don't use pymodbus with asyncio frameworks like aiohttp that rely on task cancellation.
```

Comment:
Could you possibly be more negative.

Reply:
It's pretty hard to put a positive spin on dropping support for widely used asyncio features like task cancellations IMO.

Reply:
Ok then let's close it instead.

Reply:
So it's better to keep users unaware that the library doesn't follow normal asyncio task cancellation conventions so that they get tricked into using it with incompatible applications? Our pymodbus integration is by far the most painful library integration I've ever had to maintain due to pymodbus not following typical python conventions/expectations. We have had to put wrappers around virtually every pymodbus operation in our code to get even somewhat sane exceptions being thrown.

Reply:
As I wrote earlier it is good to document conventions, but you language was not inline with the rest of the documentation and you wrote you could not write it less negative hence the close.